docs/tutorials/data_visualization.ipynb

```diff
@@ -241,36 +349,98 @@
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "titanic_numerical.plot_scatterplot(\"age\", \"fare\")\n"
    ],
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "start_time": "2023-03-14T19:00:58.343092Z",
-     "end_time": "2023-03-14T19:00:58.457212Z"
+     "start_time": "2023-06-16T08:16:58.271625Z",
+     "end_time": "2023-06-16T08:16:58.491233Z"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "The above scatterplot uses the `age` and `fare` columns from the Titanic dataset to plot each data point that represents a passenger. It can be inferred, that more than 50% of the passengers paid a fare less than 50 pounds. Three passengers aged between 35 and 60 paid a fare of about 500 pounds and were considered outliers.\n"
```

Comment:
i'd say

> Three passengers of the ages ~35 and ~60 paid a fare of about 500 pounds.

because when you say between it sounds more like they are somewhere between 35 and 60 so could also be 50 or something like that.